app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -37,7 +43,12 @@ interface VariantManager {
         val ACTIVE_VARIANTS = listOf(
             // SERP variants - do not remove
             Variant(key = "sa", weight = 1.0, features = emptyList()),
-            Variant(key = "sb", weight = 1.0, features = emptyList())
+            Variant(key = "sb", weight = 1.0, features = emptyList()),
+
+            // Light theme experiment
+            Variant(key = "mg", weight = 2.0, features = emptyList()),
```

Comment:
Any reason for the control group getting twice the size of the others? You might be reducing the more interesting datasets

Reply:
I'm performing multiple comparisons and I wanted to buckets to be the same size
* switch (mh + mj) versus no switch (mg)
* light theme as default (mh) versus dark theme as default (mj)

Reply:
When i was analysing the results of past experiments, I found trying to weight things in multiples of others pointless; it didn't make any future calculations easier as everything was always normalised anyway.

So no need to make any changes if you're happy with the setup; but it didn't seem necessary for me.

Reply:
My motivation is one of experimental design. It doesn't matter _much_ but I want an even spilt in comparison groups as it's the smallest number that determines how fast I can arrive at a statistically significant result. Happy to zoom to talk more about this.